packages/main/src/modules/ipc.ts

```diff
@@ -22,6 +22,12 @@ export function initIpc() {
 
   // bin
   handle('bin.install', () => bin.install());
+  handle('bin.installNpmPackages', (_event, projectPath, packageName) =>
+    bin.installNpmPackages(projectPath, packageName),
+  );
+  handle('bin.npmPackageOutdated', (_event, projectPath, packageName) =>
+    bin.npmPackageOutdated(projectPath, packageName),
+  );
```

Comment:
Same as above, I would move these IPCs into their own `npm` namespace: `npm.install` and `npm.outdated`